main.cpp

```diff
@@ -127,7 +137,12 @@ int main() {
         printf("\nERROR: Failed creating QSPI driver object");
         return -1;
     }
-    
+    /* Change frequency to 20MHz*/
+    if (QSPI_STATUS_OK != myQspi->set_frequency(20000000)) {
```

Comment:
Why are we setting it to 20MHz as default?

Reply:
The STM32L475 is running at 80MHz. QSPI has a prescaler that divides this frequency by 2, so we can get : 80MHz / 40MHz / 20MHz etc...
I've read somewhere in the MX25R6435F document :
'WRSR speed max. is 33MHz when issuing WRSR for performance mode switch no matter High Performance
Mode to Ultra Low Power Mode or Ultra Low Power Mode to High Performance Mode.'
That's why I've chosen 20MHz (the highest frequency below 33MHz), to make sure we won't need to change the frequency.
20MHz is also compatible with High performance mode and Ultra Low power mode of this memory.

We can try faster frequencies once we will have a mature test.